video-course/src/app/app.module.ts

```diff
@@ -16,6 +17,7 @@ import { FilterPipe } from './pipes/filter.pipe';
     BrowserModule,
     CoreModule,
     CoursesModule,
+    LoginPageModule
```

Comment:
we do not have pages, but just loginForm or something like this. This form should be part of login module - which will include logging / auth services and other related things.